packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -419,7 +419,7 @@ export class ActiveWorkflowRunner implements IWebhookManager {
 		}
 		await this.webhookService.populateCache();
 		// Save static data!
-		await WorkflowsService.saveStaticData(workflow);
+		await Container.get(WorkflowService).saveStaticData(workflow);
```

Comment:
unless there is circular dependency, this could be injected via the constructor.